Dockerfile

```diff
@@ -10,8 +10,14 @@ RUN mkdir $config_dir
 VOLUME $config_dir
 ENV CONFIG_VOLUME=$config_dir
 
+ARG use_https=''
+ENV HTTPS_ONLY=$use_https
+
+ARG whoogle_port=5000
+ENV EXPOSE_PORT=${EXPOSE_PORT:-$whoogle_port}
```

Comment:
AFAIK the use of `EXPOSE_PORT` on the RHS of this assignment isn't helping you. You effectively have an empty env at image build time, and you haven't set any ARGs or ENVs for `EXPOSE_PORT` before this line. I think it's equivalent just to do `ENV EXPOSE_PORT=$whoogle_port`

Reply:
Ah crap, yeah I was messing around with a few things and didn't mean to keep that change. Thanks for pointing that out, will remove.